src/mprismetadata.cpp

```diff
@@ -19,6 +17,210 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
 
+/*!
+    \qmltype MprisMetaData
+    \inqmlmodule Amber.Mpris
+    \brief Provides media metadata information
+
+    Provides access to information about media content.
+
+    All properties may be undefined if not applicable or known.
+    A default trackId will be returned if one is not defined.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::trackId
+    \brief Identifies the track inside a playlist
+
+    Identifier for the current track, should be a valid DBus object path.
+    Maps to MPRIS property 'mpris:trackid'.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::duration
+    \brief Length of the media in milliseconds
+
+    Maps top MPRIS property 'mpris.length'.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::coverArtUrlSmall
+    \brief Small version of a cover image for the media
+
+    An url for a cover image of the media.
+    This field is only provided for QtMultimedia compatibility, the
```

Comment:
Without fillFrom the QtMultimedia compatibility could be vague? Do we need as the target has two properties now?

Also what is small and what is large can be vague, and in that sense simpler if we wouldn't refer to sizes.

The spec also talks about "image representing the track or album" which, I guess, does not necessarily mean cover art.

Reply:
Yeah, without the fillFrom, the compatibility is not that important anymore, so maybe they could be unified.

Reply:
I'd vote for that.